packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkedForest.spec.ts

```diff
@@ -53,79 +48,58 @@ describe("ChunkedForest", () => {
 	});
 
 	it("doesn't copy data when capturing and restoring repair data", () => {
-		const initialState: JsonableTree = {
-			type: brand("Node"),
-			fields: {
-				foo: [
-					{ type: brand("Number"), value: 0 },
-					{ type: brand("Number"), value: 1 },
-					{ type: brand("Number"), value: 2 },
-				],
-			},
-		};
-		const anchors = new AnchorSet();
+		const initialState: JsonableTree = { type: brand("Node") };
```

Comment:
When authoring tests with tree data, Jenn and I discovered recently that it's a often good thing to use a non-trivial tree (i.e., a tree with a populated field). Without the field, conversion bugs between our tree represenations (which our typing doesn't always detect) can remain undetected.

Reply:
We probably don't need three child nodes though.

Reply:
This is a unit test for a very specific thing. Having tests which don't get broken by unrelated bugs is good.
Any tests that larger trees work properly should not be here as we wouldn't want such a test to be specific to this one forest implementation.

Also this test is ensuring that the underlying tree does not go through any conversion (reused the same exact object instance). It seems unlikely that a conversion could break children if the same object is uses and thus the children must remain exactly as they were.

Reply:
I do think the general guidance of at least have a test for larger data cases is good though. I just don't think this is the place to add such a test. Also I think having minimal tests and larger cases +fuzz testing is a batter approach than only having a larger test. Minimal tests are great for bug localization, debugging, and as simple examples of what the feature is, and I don't want to lose that.